app/controllers/step_elements/checklists_controller.rb

```diff
@@ -43,6 +43,7 @@ def duplicate
         @step.step_orderable_elements.where('position > ?', position).order(position: :desc).each do |element|
           element.update(position: element.position + 1)
         end
+        @checklist.name += ' (1)'
```

Comment:
This will just name the checklist the same if it's copied multiple times, so you'll have two 'Checklist (1)' checklists, named the same. Is this how it's intended to work? Shouldn't the number change to (1), (2), (3), etc. ?